test/test_cmdstan_args.py

```diff
@@ -434,41 +433,21 @@ def test_args_bad(self):
 
         with self.assertRaises(ValueError):
             # bad inits
-            cmdstan_args = CmdStanArgs(
+            CmdStanArgs(
                 model_name='bernoulli',
                 model_exe='bernoulli.exe',
                 chain_ids=[1, 2, 3, 4],
                 inits=-5,
                 method_args=sampler_args,
             )
 
-        with self.assertRaises(ValueError):
-            # bad inits
-            cmdstan_args = CmdStanArgs(
-                model_name='bernoulli',
-                model_exe='bernoulli.exe',
-                chain_ids=[1, 2, 3, 4],
-                inits='no/such/path/to.file',
-                method_args=sampler_args,
-            )
-
-        with self.assertRaises(ValueError):
```

Comment:
Was this accidentally removed? Looks like it was duplicated code and both got removed in this.

Reply:
it was triplicated code!
removed 2 copies, left one.

Reply:
whoa, I take that back - you're correct.